data/net/minecraft/client/gui/screens/worldselection/WorldSelectionList.mapping

```diff
@@ -10,9 +10,15 @@ CLASS net/minecraft/client/gui/screens/worldselection/WorldSelectionList
 		ARG 9 worldSelectionList
 	METHOD moveSelection (Lnet/minecraft/client/gui/components/AbstractSelectionList$SelectionDirection;)V
 		ARG 1 ordering
+	METHOD refreshList (Ljava/util/function/Supplier;Z)V
+		ARG 1 supplier
+		ARG 2 updateCache
 	METHOD setSelected (Lnet/minecraft/client/gui/screens/worldselection/WorldSelectionList$WorldListEntry;)V
 		ARG 1 entry
 	CLASS WorldListEntry
+		METHOD <init> (Lnet/minecraft/client/gui/screens/worldselection/WorldSelectionList;Lnet/minecraft/client/gui/screens/worldselection/WorldSelectionList;Lnet/minecraft/world/level/storage/LevelSummary;)V
+			ARG 2 worldSelectionList
+			ARG 3 levelSummary
```

Comment:
Parameter names which are assigned directly to fields should be named the same as the field.
```suggestion
		METHOD <init> (Lnet/minecraft/client/gui/screens/worldselection/WorldSelectionList;Lnet/minecraft/client/gui/screens/worldselection/WorldSelectionList;Lnet/minecraft/world/level/storage/LevelSummary;)V
			ARG 2 worldSelectionList
			ARG 3 summary
```